parsing/lexer.mll

```diff
@@ -377,7 +378,11 @@ rule token = parse
         COMMENT (s, loc) }
   | "(**"
       { let s, loc = with_comment_buffer comment lexbuf in
-        DOCSTRING (Docstrings.docstring s loc) }
+        if !handle_docstrings then
+          DOCSTRING (Docstrings.docstring s loc)
+        else
+          COMMENT (s, loc)
```

Comment:
I think @alainfrisch is worried about [MPR#6972](http://caml.inria.fr/mantis/view.php?id=6972) -- whatever you do, you should make sure that it is not an issue.